packages/blocks/src/store/private-actions.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import { processBlockType } from './process-block-type';
+
+/** @typedef {import('../api/registration').WPBlockType} WPBlockType */
+
+/**
+ * Add block type metadata to the store.
```

Comment:
There is some inconsistency in how `addBootstrappedBlock` and `getBootstrappedBlockType` are documented. I belive it should be both about the data registered on the server with `register_block_type` and alternatively on the client from `block.json` after light processing. I'm aware it's a bit confusing because of legacy considerations, but I'm happy to help clarify all nuances.

Reply:
Sounds great, I like it a lot ❤️